text/exercise1/cake.md

```diff
@@ -1,25 +1,30 @@
-## Recipice for Chocolate Cake
+## Recipe for Yellow Cake
 
-Yellow cake is delicious!  This recipie is passed down from my great-great-grandparents.
+Yellow cake is delicious!  This recipe is passed down from my great-great-grandparents.
 
-My great-great-granparents lived in Venice. They enjoyed travelling on the canals. 
+My great-great-grandparents lived in Venice. They enjoyed traveling on the canals.
 Did you know Venice became Austrian territory when Napoleon signed the Treaty of Campo 
 Formio on 12 October 1797? I did not.
 
-### Directions
-
-Grease a 9x13-inch baking pan.
-Mix the flour, baking powder, and salt together in a medium bowl.
+### Ingredients
 
-- 240 grams all-purpose flour
+- 2 cups all-purpose flour
 - 3 1/2 teaspoons baking powder
-- 1 cup salt
+- 1 teaspoon salt
 - 1 1/2 cups white sugar
 - 1/2 cup of shortening
 - 3 eggs
 - 1 cup milk 
 - 1 teaspoon vanilla extract
 
+### Directions
+
+Preheat the oven to 350 degrees F (175 degrees C).
+
+Grease a 9x13-inch baking pan.
+
+Mix the flour, baking powder, and salt together in a medium bowl.
+
 Beat sugar and shortening together in a large bowl with an electric mixer until light and fluffy.
 
 Add eggs, one at a time, beating thoroughly after each adding each egg. 
```

Comment:
You mentioned in the PR summary that you added numbered steps to the recipe, but I don't see that reflected here. Maybe consider adding numbered steps to your already well outlined and spaced directions.